aequilibrae/paths/traffic_assignment.py

```diff
@@ -576,64 +578,67 @@ def save_skims(self, matrix_name: str, which_ones="final", format="omx") -> None
         if mat_format == "omx" and not has_omx:
             raise ImportError("OpenMatrix is not available on your system")
 
-        file_name = f"{matrix_name}.{mat_format}"
-
         mats = Matrices()
-        export_name = path.join(mats.fldr, file_name)
+        for cls in self.classes:
```

Comment:
We create a skim matrix for each class in the assignment.  We might do it differently in the future if we have a more capable matrix API